### ISSUE TITLE
split terminal separation not clear enough

### ISSUE THREAD

**Issue by gsemet** [Guake/guake#1430]
I find hard to tell the limit on the terminal when splited, at least with my theme Adwaita.

Simply set the limit bigger (2 or 4 px? what about hires screens?), maybe in different color

**Comment by azasypkin** [comment]
Yeah, if one can configure color and thickness, that would be awesome! Having a color deficiency it's almost impossible to notice separator/resizer when dark themes are used.

Is there any known workaround for that @gsemet?

**Comment by tomchor** [comment]
I can definitely second that. Came here to post exactly that but found this issue.

For me and my theme (solarized dark higher contrast) I can see the vertical separations quite well but I can't see any horizontal separator.

**Comment by dxps** [comment]
Is this so difficult to implement? Sorry, I'm not a Python dev.
Besides this, can we have it available for voting at least in [FeatHub](http://feathub.com/Guake/guake)?
Thanks.

**Comment by t-pohl** [comment]
Is there any update on this? I really like the splitting functionality but for horizontal splits there seems to be no real separator.

**Comment by dxps** [comment]
I guess it also depends on the theme that you use.
Tbh, I don't even remember why I commented before about it.
Currently, I am ok with what I have (see below), with separators looking fine.

![image](https://user-images.githubusercontent.com/14140226/182022527-b150e015-e80b-4def-84ff-7cccac42b7ea.png)

**Comment by t-pohl** [comment]
I see your point. For me it looks like this:
![Screenshot from 2022-07-31 13-14-05](https://user-images.githubusercontent.com/61389866/182023604-692942c8-025c-488e-a003-7304c562c8ad.png)

I would be fine if I could make the separator line white but I have no idea how.
